packages/plugin-survey/examples/conditional_question_display.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <script src="../../jspsych/dist/index.browser.js"></script>
+  <script src="../dist/index.browser.js"></script>
+  <link rel="stylesheet" href="../../jspsych/css/jspsych.css" />
+  <link rel="stylesheet" href="../css/survey.css" />
+</head>
+
+<body></body>
+<script type="text/javascript">
+
+  const jsPsych = initJsPsych({
+    on_finish: function() {
+      jsPsych.data.displayData();
+    }
+  });
+
+  // This example shows how to make some questions conditional on previous answers
+  // from the same survey trial.
+  const vegetables = {
```

Comment:
Why does the location of the next button change in this demo? If I select neutral it is on the right, otherwise in the center.

Reply:
Ah thanks for flagging this. The reason is that there are actually 3 nav buttons at the bottom of the page: previous, next, and complete. And depending on the participants response, they are either done with the survey (selected neutral) or going on to another page. I fixed this by adding some CSS that takes the 'complete' button out of the page flow (`display: none`) when it is hidden.

Reply:
Also: this means that the buttons could potentially move on the page (if a response is made that changes the visibility of 'complete' without also removing the 'next' button), or the button could stay in the same place but the label would change between 'next' and 'complete'. I think this is still preferable to having weird previous/next button placement because of a 'complete' button that is in the page flow but hidden.